gtfs-realtime-validator-lib/src/main/java/org/mobilitydata/gtfsrtvalidator/lib/batch/BatchProcessor.java

```diff
@@ -15,23 +15,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package edu.usf.cutr.gtfsrtvalidator.lib.batch;
```

Comment:
I would leave the current package name for a few reasons:
1. It makes it much harder to look at the Git file history. GitHub isn't good with tracking changes across file renames, so you'll effectively be erasing the history.
2. This breaks any other applications depending on the library. Those projects will all have to change the package names in their classes.

Note that you can change the `<groupId>` in the pom.xml and publish it under the `org.mobilitydata` namespace without needing to change the package name in the files.

Reply:
Note we followed the same approach for gtfs-realtime-bindings - the pom.xml has the MobilityData namespace but the package name remained `com.google.transit`:
* https://github.com/MobilityData/gtfs-realtime-bindings/blob/master/java/pom.xml#L25
* https://github.com/MobilityData/gtfs-realtime-bindings/blob/master/java/src/main/java/com/google/transit/realtime/GtfsRealtime.java

Reply:
Makes total sense, thanks!